### ISSUE TITLE
Type Around feature is missing aria tags to be read properly by screen-readers

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor5#11936]
## 📝 Provide detailed reproduction steps (if any)

Based on support ticket np. 75352. Accessibility concern on SVG icons. The svg icons that are used in buttons should have an `alt` or `aria-label` or `role` tag(s) to be AA compliment.

### ✔️ Expected result

Names of buttons for Type Around feature should be discoverable by screen readers.

### ❌ Actual result

Names of buttons for Type Around feature are not discoverable by screen readers.

### ❓ Possible solution

_If you have ideas, you can list them here. Otherwise, you can delete this section._

## 📃 Other details

* Browser: Any
* OS: Any
* First affected CKEditor version: N/A
* Installed CKEditor plugins: widget

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by zutler** [comment]
Is it possible to increase the priority of this issue?
Our application fails the ADA compliance scan, which is mandatory by the US Law.
Please provide the estimated date when this issue will be addressed.

**Comment by zutler** [comment]
Hi, 
I wanted to follow up on this...
Could you provide the release version which will include the ADA fixes?

**Comment by Comandeer** [comment]
I can confirm that labels are missing for the buttons. However, these buttons aren't focusable via keyboard. Probably the better solution would be to:

*   hide these buttons completely from screen readers using `[aria-hidden=true]`,
*   add a description (via `[aria-describedby]`) to a whole widget with information that the user can type around it by pressing Enter or Shift+Enter.
